Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -32,40 +47,82 @@ open class WayNameView: UIView {
         }
     }
     
+    open override var layer: CALayer {
+        containerView.layer
+    }
+    
+    /**
+     The background color of the `WayNameView`.
+     */
+    @objc dynamic public override var backgroundColor: UIColor? {
+        get {
+            containerView.backgroundColor
+        }
+        
+        set {
+            containerView.backgroundColor = newValue
+        }
+    }
+    
+    /**
+     The color of the `WayNameView`'s border.
+     */
     @objc dynamic public var borderColor: UIColor? {
         get {
             guard let color = layer.borderColor else { return nil }
             return UIColor(cgColor: color)
         }
+        
         set {
             layer.borderColor = newValue?.cgColor
         }
     }
     
+    /**
+     The width of the `WayNameView`'s border.
+     */
+    @objc dynamic public var borderWidth: CGFloat {
+        get {
+            layer.borderWidth
+        }
+        
+        set {
+            layer.borderWidth = newValue
+        }
+    }
+    
     public override init(frame: CGRect) {
         super.init(frame: frame)
         commonInit()
     }
     
-    public required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
+    public required init?(coder decoder: NSCoder) {
+        super.init(coder: decoder)
         commonInit()
     }
     
     func commonInit() {
```

Comment:
not relevant to this PR, but makes sense to change access level for commmonInit to private as far as I see.